conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -11,3 +11,7 @@
 
 # Add all the application routes to the prod.routes file
 ->        /        prod.Routes
+
+DELETE        /emcstfereferencedata/test-only/codelists                      uk.gov.hmrc.emcstfereferencedata.controllers.testonly.TestOnlyController.deleteCodeLists()
+DELETE        /emcstfereferencedata/test-only/deleteexciseproducts           uk.gov.hmrc.emcstfereferencedata.controllers.testonly.TestOnlyController.deleteExciseProducts()
+DELETE        /emcstfereferencedata/test-only/deletecncodes                  uk.gov.hmrc.emcstfereferencedata.controllers.testonly.TestOnlyController.deleteCnCodes()
```

Comment:
```suggestion
DELETE        /emcs-tfe-reference-data/test-only/codelists                      uk.gov.hmrc.emcstfereferencedata.controllers.testonly.TestOnlyController.deleteCodeLists()
DELETE        /emcs-tfe-reference-data/test-only/excise-products           uk.gov.hmrc.emcstfereferencedata.controllers.testonly.TestOnlyController.deleteExciseProducts()
DELETE        /emcs-tfe-reference-data/test-only/cn-codes                  uk.gov.hmrc.emcstfereferencedata.controllers.testonly.TestOnlyController.deleteCnCodes()
```

Reply:
A couple of suggestions here:
* We should use the same context path as the rest of the service `emcs-tfe-reference-data` - you can find this in `prod.routes`
* The method is already a `DELETE` so there is no need to repeat that in the path